### DISCUSSION TITLE
Is there a way to format file paths as just their basenames? (Like with `find -printf`.)

### DISCUSSION THREAD

**Discussion by jcfk** [BurntSushi/ripgrep#2589] [Q&A] [answered]
In particular is there a way to output just basenames? For example `file.md:2:match` as opposed to `path/to/file.md:2:match`. Great tool, thanks a lot.

**Comment by BurntSushi** [comment, marked as answer]
No, there isn't. I think your best bet is some kind of post-processing step on ripgrep's output.
